articles/architecture-scenarios/implementation/b2c/branding.md

```diff
@@ -0,0 +1,112 @@
+
+::: warning
+Exercise caution regarding the use of third-party JavaScript on your Login Page, since sensitive security-related information often flows through the page and the introduction of cross-site scripting (XSS) vulnerabilities is of particular concern.
+:::
+
+The Login Page comes with [multi-language support](/libraries/lock/v11/i18n) provided by default. In our experience, almost all applications need internationalization/localization, and while it is fairly simple to add during the development phases, it can become difficult at a later stage. As such, we recommend, at the very least, accounting for this need if you do not implement either feature during this phase.
+
+::: panel Best Practice
+For those who want to build and use a fully-customized user interface using technologies like React or Angular.js, the Auth0 JavaScript SDK ([auth0js](/libraries/auth0js) allows you to add Auth0 functionality to your custom login page.
+:::
+
+## Change password page customization
+
+The Change Password page is used whenever a user takes advantage of the [password reset](/universal-login/password-reset) functionality. Like the login page, you can customize the Change Password page to reflect your organization's branding. 
+
+## Error page customization
+
+If there are issues with user sign up or login, Auth0 provides error messages that indicate what the problem is under the hood. The default messages are somewhat cryptic, especially to the end user, since they will likely be missing context that only you can supply. As such, we recommend customizing your error pages to provide the missing context-specific information directly to your users.
+
+Furthermore, customizing your error pages allows you to display your branding, not Auth0's, as well as provide useful information to your users as to what should be done next. This information might include a link to a FAQ or how to get in touch with your company's support team or help desk.
+
+At the minimum, we recommend that you provide your visitors with:
+
+* Visual indicators like a friendly-name and your logo for display
+* An email address and URL for your organization's Support team
+
+Out-of-the-box there is no user interface for customizing your error pages, but you can use the [Tenant Settings endpoint of the Management API](/api/management/v2#!/Tenants/patch_settings) to configure them. Alternatively, if you can create and host your error page then you can have Auth0 direct users to that page instead of using the Auth0-hosted option. 
+
+## Custom domain naming
+
+By default, the URL associated with your tenant will include its name and possibly a region-specific identifier. For example, tenants based in the US have the URL **example.auth0.com** while those based in Europe have **example.eu.auth0.com**.
+
+If you want to use a name that is consistent with your company's brand, you can configure a [custom domain](/custom-domains). Each tenant can support one custom domain name.
```

Comment:
This was originally provided via an attention callout (https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.600emxp0rxpf). It's important to know that you can only have one custom domain per Auth0 tenant, so drawing attention to this fact would I feel be better 👍

Reply:
Done